metabase/metabase
# Fix SDK interactive question showing no question error incorrectly

Closes https://github.com/metabase/metabase/issues/42352

### Description

The problem happened because we only relied on `loading` which indicates whether the initial question has been loaded or not, after that there could be another loading state when users drills down.

### How to verify

1. Using Customer Zero app.
1. Go to Products > click on a product > Drills on the first bar chart. And there should be no question error state anymore.
1. Click other page on the nav bar e.g. kitchen sink.
1. Click Products again > click on a probuct and there should be no question error state.

### Demo
#### Before

https://github.com/metabase/metabase/assets/1937582/79cd60ef-60d2-471d-aeac-f0ede2e3299d


#### After
https://www.loom.com/share/65437958d8ff461796e73de630a763b6

### Checklist
